tequilapi/endpoints/connection_test.go

```diff
@@ -34,7 +34,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-type fakeManager struct {
+type fakeConnectionManager struct {
```

Comment:
fakeConnectionManager -> mockConnectionManager. We probably should not use the `fake` anymore.

Reply:
renamed